docs/pages/getting-started/migrating-to-v5.mdx

```diff
@@ -34,7 +34,7 @@ npm install next-auth@beta
 #### Universal `auth()`
 
 - A single method to authenticate anywhere
-- Use `auth()` instead of `getServerSession`, `getSession`, `withAuth`, `getToken`, and `useSession` ([Read more](#authenticating-server-side))
+- Use `auth()` instead of `getServerSession`, `getSession`, `withAuth`, `getToken` ([Read more](#authenticating-server-side))
```

Comment:
```suggestion
- Use `auth()` instead of `getServerSession`, `getSession`, `withAuth`, `getToken`, and `useSession` ([Read more](#authenticating-server-side))
```